src/components/WikiResults.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+
+import Spinner from './Spinner';
+import ErrorMessage from './ErrorMessage';
+
+import Map from './Map';
+
+const WikiResults = () => {
```

Comment:
I believe I saw the WikiResults component before 😄. I'm not sure, but I guess you will merge it after.